common/config/rush/build-cache.json

```diff
@@ -0,0 +1,130 @@
+/**
+ * This configuration file manages Rush's build cache feature.
+ * More documentation is available on the Rush website: https://rushjs.io
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/build-cache.schema.json",
+
+  /**
+   * (Required) EXPERIMENTAL - Set this to true to enable the build cache feature.
+   *
+   * See https://rushjs.io/pages/maintainer/build_cache/ for details about this experimental feature.
+   */
+  "buildCacheEnabled": false,
```

Comment:
Why not enable it?

Reply:
This monorepo only has 5 projects, so there wouldn't be much benefit. It seems useful to maintain a public production monorepo with different settings from the official Rush Stack repo